client/src/mock-data/mockFormulas.js

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+const mockFormulas = [
+  {
+    id: 1,
+    name: 'Weighted average',
+    attributes: ['Max Points', 'Min required points', 'Weight'],
+    codeSnippet: `
+    const weightedAverage = (nums, weights) => {
```

Comment:
The code snippet could be written like this so that it is aligned to the center:
```
codeSnippet: 
    `
    const weightedAverage = (nums, weights) => {
      const [sum, weightSum] = weights.reduce(
        (acc, w, i) => {
          acc[0] = acc[0] + nums[i] * w;
          acc[1] = acc[1] + w;
          return acc;
        },
        [0, 0]
      );
      return sum / weightSum;
     };
    `,
```